notebooks/spark/AerospikeSparkPython.ipynb

```diff
@@ -224,9 +224,9 @@
       "+---+----+\n",
       "| 10|null|\n",
       "| 50|null|\n",
+      "|185|null|\n",
```

Comment:
Change "Insert data using sql insert staements" --> "Using Spark SQL syntax to insert data"